.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: Release with SLSA provenance
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - 'release/**'
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write # Required for SLSA provenance generation
+
+jobs:
+  build-and-provenance:
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        
+      - name: Set up Docker Buildx
```

from SLSA_Security_Expert
docker/setup-buildx-action も固定のSHAハッシュを使用すべきです。

```suggestion
        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
```